scripts/library.sh

```diff
@@ -339,6 +339,12 @@ function start_knative_serving() {
   wait_until_pods_running knative-serving || return 1
 }
 
+# Install the stable release Knative/serving in the current cluster.
+# Parameters: $1 - Knative Serving version number or latest.
+function start_release_knative_serving() {
+  start_knative_serving "https://github.com/knative/serving/releases/download/$1/serving.yaml"
```

Comment:
For consistency, the URL should be

```suggestion
  start_knative_serving "https://storage.googleapis.com/knative-releases/serving/previous/v$1/serving.yaml"
```

Reply:
done